app/helpers/evaluations_helper.rb

```diff
@@ -29,24 +29,24 @@ def display_score(assignment)
   end
 
   # individual evaluator score
-  def evaluator_score(assignment)
+  def evaluator_score(assignment, format: :zero)
     score = display_score(assignment)
-    return Score.new(0, "0", "N/A") if score == 'N/A'
+    return Score.new(0, format == :zero ? "0" : "-", "-") if score == 'N/A'
```

Comment:
yikes i didn't realize you applied the same change on multiple PRs, that will be an issue now 😕